packages/icons/src/icon-product/air-gapped-inverse.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>icon/system/air-gapped</title>
+    <defs>
+        <path d="M2.375,3.9995 L4.625,3.9995 L4.625,2.8125 C4.625,2.19117966 4.12132034,1.6875 3.5,1.6875 C2.87867966,1.6875 2.375,2.19117966 2.375,2.8125 L2.375,3.9995 Z M1.25519081,2.65845119 C1.33435508,1.48766159 2.30913601,0.5625 3.5,0.5625 C4.74264069,0.5625 5.75,1.56985931 5.75,2.8125 L5.75,2.8125 L5.75,3.9995 L5.99707067,4 C6.55097324,4 7,4.44792891 7,4.99869585 L7,7.50130415 C7,8.05286864 6.56211865,8.5 5.99707067,8.5 L1.00292933,8.5 C0.449026756,8.5 0,8.05207109 0,7.50130415 L0,4.99869585 C0,4.44713136 0.437881351,4 1.00292933,4 L1.25,3.9995 L1.25,2.8125 Z" id="path-1"></path>
+    </defs>
+    <g id="icon/system/air-gapped" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M14,2 L14,7 L11.832597,6.99939356 C11.4595757,6.72476992 10.9987379,6.5625 10.5,6.5625 C10.0011002,6.5625 9.54012511,6.72487531 9.16703968,6.99966109 L2,7 L2,2 L14,2 Z M11,3.75 L7.75,3.75 L7.64822944,3.75684662 C7.28215388,3.80650904 7,4.12030423 7,4.5 C7,4.87969577 7.28215388,5.19349096 7.64822944,5.24315338 L7.75,5.25 L11,5.25 L11.1017706,5.24315338 C11.4678461,5.19349096 11.75,4.87969577 11.75,4.5 C11.75,4.08578644 11.4142136,3.75 11,3.75 Z M5,3.75 C4.58578644,3.75 4.25,4.08578644 4.25,4.5 C4.25,4.91421356 4.58578644,5.25 5,5.25 C5.41421356,5.25 5.75,4.91421356 5.75,4.5 C5.75,4.08578644 5.41421356,3.75 5,3.75 Z" id="Combined-Shape" fill="#FFF" opacity="0.4"></path>
+        <path d="M8.40141121,7.9993897 C8.32128891,8.20603705 8.2708093,8.42746369 8.25519081,8.65845119 L8.25519081,8.65845119 L8.25,8.8125 L8.25,9.74935983 L7.75,9.75 L7.64822944,9.75684662 C7.28215388,9.80650904 7,10.1203042 7,10.5 C7,10.5806666 7.01273509,10.6583587 7.03630441,10.7311756 C7.0124547,10.8156227 7,10.9056741 7,10.9986959 L7,10.9986959 L7,12.9993598 L2,13 L2,8 Z M14,10.9986959 C14,10.4479289 13.5509732,10 12.9970707,10 L12.9970707,10 L12.75,9.9995 L12.75,8.8125 L12.7448092,8.65845119 C12.7291687,8.4271379 12.6785686,8.20541246 12.5982496,7.99851542 L14,8 Z M5,9.75 C4.58578644,9.75 4.25,10.0857864 4.25,10.5 C4.25,10.9142136 4.58578644,11.25 5,11.25 C5.41421356,11.25 5.75,10.9142136 5.75,10.5 C5.75,10.0857864 5.41421356,9.75 5,9.75 Z M11.2764053,7.99835983 C11.4911795,8.20323985 11.625,8.4922422 11.625,8.8125 L11.625,8.8125 L11.625,9.9995 L11.5578459,9.99868391 C11.4205539,9.8460107 11.2214892,9.75 11,9.75 L9.375,9.74935983 L9.375,8.8125 C9.375,8.4922422 9.50882052,8.20323985 9.72359468,7.99835983 Z" id="Combined-Shape" fill="#FFF" opacity="0.4"></path>
+        <g id="icon/system/lock" transform="translate(7.000000, 6.000000)">
+            <mask id="mask-2" fill="white">
```

Comment:
wondering about using a fill here. What happens if it's used on a gray background?

Reply:
I think that is why we have two. One with colors and one that is white. TBH when I got them one was blue and the other was dark purple but they didn't match our color scheme so I made one white.

Reply:
yeah you are definitely following the existing convention here, so I'll table my concern